go/libkb/sig_chain.go

```diff
@@ -106,6 +106,16 @@ func last(links []*ChainLink) (ret *ChainLink) {
 	return links[len(links)-1]
 }
 
+func (sc *SigChain) ChainLinksCopy() []ChainLink {
+	links := make([]ChainLink, len(sc.chainLinks))
+	for i := 0; i < len(sc.chainLinks); i++ {
+		links[i] = *sc.chainLinks[i]
```

Comment:
Does this need to be a deep copy?  `ChainLink` itself has a bunch of pointers dangling off of it.